sklearn/tests/test_impute.py

```diff
@@ -572,6 +572,24 @@ def test_iterative_imputer_predictors(predictor):
     assert len(set(hashes)) == len(hashes)
 
 
+def test_iterative_imputer_bayesianridge_default():
```

Comment:
Would it be better to just make this case in the previous test

Reply:
I meant:

```python
@pytest.mark.parametrize(
    "predictor",
    [None, DummyRegressor(), BayesianRidge(), ARDRegression(), RidgeCV()]
)
def test_iterative_imputer_predictors(predictor):
    rng = np.random.RandomState(0)
    n = 100
    d = 10
    X = sparse_random_matrix(n, d, density=0.10, random_state=rng).toarray()
    imputer = IterativeImputer(missing_values=0,
                               n_iter=1,
                               predictor=predictor,
                               random_state=rng)
    imputer.fit_transform(X)
    # check that types are correct for predictors
    hashes = []
    for triplet in imputer.imputation_sequence_:
        expected_type = type(predictor) if predictor is not None else type(BayesianRidge())
        assert isinstance(triplet.predictor, expected_type)
        hashes.append(id(triplet.predictor))
    # check that each predictor is unique
    assert len(set(hashes)) == len(hashes)
```

Reply:
Good idea. Done.